Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -3815,5 +3816,207 @@ def dict_safe_get(dict_object, keys, default_return_value = None):
     return dict_object
 
 
+if IS_PY3:  # merge_lists does not work in python 2
+    CONTEXT_UPDATE_RETRY_TIMES = 3
+    MIN_VERSION_FOR_VERSIONED_CONTEXT = '6.0.0'
+
+    def merge_lists(original_list, updated_list, key):
+        """
+        Replace values in a list with those in an updated list.
+
+        :type original_list: ``list``
+        :param original_list: The original list.
+
+        :type updated_list: ``list``
+        :param updated_list: The updated list.
+
+        :type key: ``str``
+        :param key: The key to replace elements by.
+
+        :rtype: ``list``
+        :return: The merged list.
+
+        Example:
```

Comment:
need to check how this looks in the script helper, i think we might need to move the examples to be above the args